sources/packages/web/src/views/aest/SupportingUser.vue

```diff
@@ -8,13 +8,19 @@
     subTitle="Financial Aid Application"
   >
   </header-navigator>
+  <application-header-title :application-id="applicationId" />
+  <div v-if="formData && formData.parentFullName">
```

Comment:
Is there a reason to create a `div`?
If not, please move the `v-if="formData && formData.parentFullName"` into the `detail-header`.